elpy.el

```diff
@@ -2050,14 +2063,19 @@ This requires Python 2.7 or later."
 (put 'elpy-test-discover-runner 'elpy-test-runner-p t)
 
 (defun elpy-test-django-runner (top file module test)
-  "Test the project using the Django discover runner.
+  "Test the project using the Django discover runner,
+   or with manage.py if elpy-test-django-with-manage is true.
 
 This requires Django 1.6 or the django-discover-runner package."
   (interactive (elpy-test-at-point))
   (if module
       (apply #'elpy-test-run
-             top
-             (append elpy-test-django-runner-command
+             (if elpy-test-django-with-manage
+                 (expand-file-name (locate-dominating-file (elpy-project-root) "manage.py"))
+               top)
+             (append (if elpy-test-django-with-manage
+                         elpy-test-django-runner-manage-command
+                       elpy-test-django-runner-command)
                      (list (if test
```

Comment:
- The _directory_ in which to run `elpy-test-run` should *probably* be the directory of `manage.py`, no?
- The _command_ to run should be the full path to manage.py